src/iris/bin/sender.py

```diff
@@ -1462,6 +1474,12 @@ def message_send_enqueue(message):
 
     message_mode = message.get('mode')
     if message_mode and message_mode in per_mode_send_queues:
+        # check for known corner case limitations and correct accordingly
+        if message['mode'] == 'call':
+            message_mode, message_body = check_call_restrictions(message)
+            message['mode'] = message_mode
+            message['body'] = message_body
```

Comment:
Are we overwriting the message here, or appending to the body? I'd prefer adding a blurb at the beginning/end of the message, rather than replacing the whole text.

Reply:
The message body gets appended later so we're not overwriting the message, just prepending this warning. For extra robustness/futueproofness I added an extra check so that we append any existing message body.